cdap-ui/app/directives/widget-container/widget-factory.js

```diff
@@ -275,10 +275,12 @@ angular.module(PKG.name + '.commons')
         }
       },
       'multi-select': {
-        element: '<my-multi-select-dropdown></my-multi-select-dropdown>',
+        element: '<multi-select></multi-select>',
         attributes: {
-          'ng-model': 'model',
-          'config': 'myconfig'
+          'value': 'model || myconfig.properties.defaultValue || [myconfig["widget-attributes"].defaultValue]',
+          'options': '(myconfig.properties.options || myconfig["widget-attributes"].options)',
```

Comment:
you can remove the `myconfig.properties.options`. We'll enforce the options to be under widget-attributes